### DISCUSSION TITLE
How to navigate on the Akvorado UI to read Netflow and SFlow data?

### DISCUSSION THREAD

**Discussion by agust-d23** [akvorado/akvorado#1318] [Q&A]
Hello there! I have recently installed Akvorado on my VM (Alma linux OS) and wanted to start getting some Netflow data to show on UI. I am able to access the UI successfully. I can see some synthetic flows are generated in the background, and I was trying to navigate to find how I can view netflow information(just playing around trying to familiarize myself). But since this is very new to me unfortunately I couldn’t find specific steps on how to navigate the Akvorado UI to read Netflow data. Could someone please help me with this , Highly appreciate any input/help provided since i am a newbie pls ignore my sillyness i am just trying to learn and figure this out :) thankyou!

**Comment by vincentbernat** [comment]
Tab "[Visualize](https://demo.akvorado.net/visualize/)". If you don't have graphs, you need to configure interface classification (as explained in the quick start documentation).
